public/embed.js

```diff
@@ -1,24 +1,24 @@
 // @ts-check
 
-(function() {
+(function () {
   'use strict';
 
   /**
    * @param {() => void} loaded
    */
-  var ready = function(loaded) {
-    if (['interactive', 'complete'].indexOf(document.readyState) !== -1) {
-      loaded();
-    } else {
-      document.addEventListener('DOMContentLoaded', loaded);
-    }
+  var ready = function (loaded) {
+    document.onreadystatechange = function () {
+      if (document.readyState === 'complete') {
+        loaded();
+      }
+    };
```

Comment:
Wouldn't the following be better?

```suggestion
    document.addEventListener('readystatechange', function () {
      if (document.readyState === 'complete') {
        loaded();
      }
    });
```